src/api.v2/helpers/access/createUserInvite.js

```diff
@@ -0,0 +1,40 @@
+const UserAccess = require('../../model/UserAccess');
+
+const { getAwsUserAttributesByEmail } = require('../../../utils/aws/user');
+const sendEmail = require('../../../utils/send-email');
```

Comment:
For `send-email`: this is not named according to our naming convention. Please change to camel case to match the majority of other files.

Reply:
changed